app/helpers/sessions_helper.rb

```diff
@@ -28,6 +28,10 @@ def current_user
 		end
 	end
 
+	def check_or_set_user
+		@user ||= current_user
```

Comment:
My indentation looks good but i still tried to double-check. Here: 298566ecef29c5bf397e1cea5786dcb1eadc6fff